torch/csrc/Device.cpp

```diff
@@ -191,7 +191,6 @@ PyTypeObject THPDeviceType = {
   sizeof(THPDevice),                     /* tp_basicsize */
   0,                                     /* tp_itemsize */
   nullptr,                               /* tp_dealloc */
-  // NOLINTNEXTLINE(modernize-use-nullptr)
   0,                                     /* tp_vectorcall_offset */
```

Comment:
Item after `tp_dealloc` is `tp_print`, isn't it?
https://github.com/python/cpython/blob/0a0a135bae2692d069b18d2d590397fbe0a0d39a/Include/object.h#L353-L354
```suggestion
  nullptr,                                     /* tp_print */
```